packages/dashboard/src/components/layout/MainNavbar/MainNavbar.js

```diff
@@ -1,32 +1,132 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { withRouter } from 'react-router'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
-import { Container, Navbar } from 'shards-react'
+import { Container, Navbar, Row, Col } from 'shards-react'
+
+import { tradeActions } from '../../../redux/actions'
+import { searchSecurity } from '../../../utils/requests'
 
 import NavbarSearch from './NavbarSearch'
 import NavbarNav from './NavbarNav/NavbarNav'
 import NavbarToggle from './NavbarToggle'
 
-const MainNavbar = ({ layout, stickyTop }) => {
+const DropDownItem = ({
+  name,
+  symbol,
+  exchange,
+  id,
+  currency,
+  selectSecurity,
+}) => {
+  return (
+    <div
+      className="securityDropDownItem"
+      onClick={() => selectSecurity({ id, name, symbol, exchange, currency })}
```

Comment:
I'm a little confused why you have this onclick and also the onDropdownSecurityClick that both fire the selectSecurity action (also with different parameters here)

Reply:
Agreed the naming is a little confusing, this should probably be more generalized or keep the same `onDropDownSecurityClick` name.

This is not actually an action creator, but just the handler you referenced above. Will rename this.